.pre-commit-config.yaml

```diff
@@ -1,19 +1,19 @@
 repos:
   -   repo: https://github.com/pycqa/isort
-      rev: 5.9.3
+      rev: 5.10.1
       hooks:
       - id: isort
         language_version: python3
   - repo: https://github.com/asottile/pyupgrade
     # Do not upgrade: there's a bug in Cython that causes sum(... for ...) to fail;
     # it needs sum([... for ...])
-    rev: v2.13.0
+    rev: v2.31.0
```

Comment:
I think this hasn't been resolved, yet. All the cython builds fail due to this. I think the error is different than last time but its' definitely linked to cythonization

Reply:
Whoops, I totally missed that comment about not updating `pyupgrade`. Just updated this PR accordingly